src/components/Layout/index.tsx

```diff
@@ -1,9 +1,9 @@
 import { FC, ReactNode } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import Image from 'next/image';
 import Footer from '@/components/footer';
 import styles from '@/components/Layout/Layout.module.scss';
+import NavBar from '../navBar';
```

Comment:
Please use the absolute imports here, refer above imports. 
https://nextjs.org/docs/advanced-features/module-path-aliases